src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -112,11 +128,16 @@ class MetadataBasedItemList extends React.Component<Props, State> {
 
     cellRenderer = ({ columnIndex, rowIndex, key, style }: CellRendererArgs): Element<'div'> => {
         const { hoveredRowIndex } = this.state;
-        const data = rowIndex === 0 ? this.getGridHeaderData(columnIndex) : this.getGridCellData(columnIndex, rowIndex);
+        const isFileNameCell = rowIndex > HEADER_ROW_INDEX && columnIndex === FILE_NAME_COLUMN_INDEX;
+        const data =
+            rowIndex === HEADER_ROW_INDEX
```

Comment:
This is getting a bit difficult to read and repetitive. Can we introduce some additional helper variables, like `isHeader` and `isFileIconCell`, that can also be used by the class names code below?